hveto/const.py

```diff
@@ -0,0 +1,33 @@
+import os
+from gwpy.time import from_gps
+
+
+def get_hvetopath(gpstime):
+
+    """ Returns the path of hveto output files
+
+    Given a gpstime, the path of the folder
+    containing hveto trigger files is returned
+
+    Parameters
+    ----------
+    gpstime : `str` or `float`
+    start time of the day for this analysis
+
+    Returns
+    _______
+    path : `str`
+
+    Example
+    _______
+
+    >>> from hveto.const import get_hvetopath
+    >>> get_hvetopath(1257811218)
+    '/home/detchar/public_html/hveto/day/20191115/latest'
+    """
+
+    date = from_gps(gpstime).strftime('%Y%m%d')
+    hveto_dir = '/home/detchar/public_html/hveto/day/'
```

Comment:
Is it worth making this a global variable? Whether or not we do, it's definitely worth leaving a comment that this path assumes the production DetChar workflow on the LHO and LLO computing clusters

Reply:
Yeah, I can add a comment for this.